packages/profile-picture/src/profile-picture.ts

```diff
@@ -24,16 +23,18 @@ export class ProfilePictureElement extends LitElement {
    */
   @property({ type: Number }) size: number = 120;
 
+  @property({ type: Boolean }) private hasError: boolean = false;
+
   @property({ type: Number }) private cacheBust: number = 0;
 
-  private _availableSizes = new Set([32, 64, 128, 256, 512, 1024]);
+  private _availableSizes = new Set([32, 64, 128, 256, 512]);
```

Comment:
No 1024? We are creating a 1024 size right?

Reply:
We are not.

Reply:
Do we need 1024?